server/server.js

```diff
@@ -24,10 +27,63 @@ app.get("/api/rentals", (req, res) => {
     });
 });
 
-app.get("/app.js", (req, res) => {
-  res.sendFile(path.resolve(__dirname, "../public/bundle.js"));
+app.delete("/api/rentals", (req, res) => {
+  const id = parseInt(req.query.id);
+
+  Rental.deleteOne({ _id: id })
+    .then(result => {
+      let message;
+      if (result.deletedCount === 0) {
```

Comment:
I like the informative responses

Reply:
I'm trying out some different things. I appreciate the feedback.